rclcpp/include/rclcpp/client.hpp

```diff
@@ -735,6 +812,11 @@ class Client : public ClientBase
       CallbackInfoVariant>>
   pending_requests_;
   std::mutex pending_requests_mutex_;
+
+private:
+  using ClientIntraProcessT = rclcpp::experimental::ClientIntraProcess<ServiceT>;
+  std::shared_ptr<ClientIntraProcessT> client_intra_process_;
+  std::atomic_uint sequence_number_{1};
```

Comment:
should this be named "ipc_sequence_number_" ?

Reply:
Done in 3e4d9c65